### ISSUE TITLE
Bug: Single Job - Preview does not display variable information

### ISSUE THREAD

**Issue by em-herrick** [GSA/notifications-admin#1373]
### Who discovered this?

Washington, in user research session

### What happened?

A bug happened! Sending one off messages with variable information:

1. ` preview` does not fill in variable information (bug)
2. User mentions that `select delivery status` feels redundant to `preview` screen is redundant since there is no send list (more than a bug, but wanted to mention)

https://github.com/GSA/notifications-admin/assets/91496588/00d4bbf3-2ae8-46b6-a228-d2d2551fb778

### Environment

Production

### What pages did this appear on?

Send flow for a single message job with variable information. The staff that send messages in Washington send to one person at a time. They input variable information not via spreadsheet but via the user interface.

### Detail the steps for someone to reproduce

1. Select a template with variable information
2. Send to one phone number
3. Input variable information via single send interface
4. View message on `Select delivery time` _- variable information appears_.
5. Select send "now" 
6. View message on `Preview message` _-variable message does not appear AND user mentions that this page is a redundant since there is no send list. In the future, could we remove this second preview page for messages that don't have a send list?_

### What browsers are you seeing the problem on?

Microsoft Edge

**Comment by em-herrick** [comment]
In a user research session, we found a bug on the preview page for a single send with variable info. @heyitsmebev I assigned to you since you've been working on this section.

One thing I am wondering about, is previously (perhaps before we implemented the USWDS) all of the variable information use to be highlighted in yellow, so it was easier to see when scanning. Is this something that would be easy to bring back while looking into this bug? 

![Image](https://github.com/GSA/notifications-admin/assets/91496588/46ebf267-301a-40b1-8a12-9dc738db360b)
